doc/courses/sosp/application.md

```diff
@@ -192,10 +192,13 @@ we can use Tock to provide the BLE
 [Environmental Sensing Service](https://www.bluetooth.com/specifications/assigned-numbers/environmental-sensing-service-characteristics)
 (ESS).
 
-Currently, the Tock libraries do not support directly interacting with the BLE
-radio. However, we can still access BLE by loading an additional process on the
-board as a service and sending it commands over Tock's inter-process
-communication (IPC) mechanism.
+Tock implements raw BLE functionality in kernelspace and exposes this to
```

Comment:
I'd be a little careful with this sentence. We don't really implement any BLE functionality in the kernel for SAM4L-based boards. Instead we run an application using Nordic libraries and it ships everything to the nRF over UART.

Reply:
Good point. Should it say "exposes BLE functionality from kernelspace" instead, leaving the question of who actually implemented BLE unanswered?